dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -184,6 +184,10 @@
     @ApiModelProperty(value = "The Id of the corresponding topic on Dockstore Discuss")
     private Long topicId;
 
+    @Column(name = "next_version_name")
+    @ApiModelProperty(value = "Name of the next version (applicable to hosted only)")
+    private String nextVersionName;
```

Comment:
Any reason why string vs integer?

Reply:
version names are all stored as strings.  in the event we're doing some other weirding naming convention that is not just 1,2,3,4, etc.  this does not need to be changed

Reply:
FYI, I had created a feature request #2158 to let users specify their own versions. Saw this comment before I reviewed the code, and not sure if matters, e.g., if we do it, maybe we'll store the user-specified version in a different column, but just wanted to mention it in case it brings up any issues that I can't think of right now.

Reply:
I don't see how that solves this solves the problem if people can arbitrarily name their versions though.  If a hosted workflow has versions `a`,`b`,`spam`,and`chicken` then they delete `spam` and re-create `spam`, its not clear to me how storing a single value in a column helps. 

For that matter, if the naming schema is arbitrary, we actually can't predict the name of the next version. Maybe this needs to be storing a blacklist instead

Reply:
I was just raising the possibility of a potential future issue, not proposing it as a solution to this issue. And you thought of a good use case that I hadn't where there really would be a problem...

That said, it hasn't even decided to add the user-specified version feature, so we could cross the bridge when/if we get there.